<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IpSettingsDialog</class>
 <widget class="QDialog" name="IpSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IpSettingsDialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/L2tpIPsecVpn.qrc">
    <normaloff>:/images/configure.png</normaloff>:/images/configure.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="m_pDialogLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="m_pFrame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="m_pFrameLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QCheckBox" name="m_pAutomaticDNSCheckBox">
        <property name="toolTip">
         <string>If checked the peer is asked for up to 2 DNS server addresses, otherwise provide the desired primary and secundary DNS server addresses manually in the edit boxes below.</string>
        </property>
        <property name="text">
         <string>Obtain DNS server addresses automatically</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="m_pPrimaryDNSServersLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Preferred DNS server:</string>
        </property>
        <property name="buddy">
         <cstring>m_pPrimaryDNSServersEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="m_pPrimaryDNSServersEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Ip addresses of preferred domain name server used to resolve host names. Address must be entered as IPv4 address.

Example:
1.2.3.4</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="m_pSecundaryDNSServersLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Alternate DNS server:</string>
        </property>
        <property name="buddy">
         <cstring>m_pSecundaryDNSServersEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="m_pSecundaryDNSServersEdit">
        <property name="toolTip">
         <string>Ip addresses of alternate domain name server used to resolve host names. Address must be entered as IPv4 address.

Example:
1.2.3.4</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="m_pSearchDomainsLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Search domains:</string>
        </property>
        <property name="buddy">
         <cstring>m_pSearchDomainsEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="m_pSearchDomainsEdit">
        <property name="toolTip">
         <string>Domains used when resolving hostnames.
Use spaces to separate multiple domains.</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="m_pRoutesButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>27</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Routes ...</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <spacer name="m_pVerticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>23</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="m_pButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_pAutomaticDNSCheckBox</tabstop>
  <tabstop>m_pPrimaryDNSServersEdit</tabstop>
  <tabstop>m_pSecundaryDNSServersEdit</tabstop>
  <tabstop>m_pSearchDomainsEdit</tabstop>
  <tabstop>m_pRoutesButton</tabstop>
  <tabstop>m_pButtonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/L2tpIPsecVpn.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>IpSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>IpSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pAutomaticDNSCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pPrimaryDNSServersEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pAutomaticDNSCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pSecundaryDNSServersEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pAutomaticDNSCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pSearchDomainsEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
